server/routes.js

```diff
@@ -2,8 +2,34 @@ const { initManagers } = require('./db/init_db.js');
 // eslint-disable-next-line no-unused-vars
 const { postManager, userManager } = initManagers();
 module.exports = function (app) {
-	app.post('/posts', function (req, res) {
-		console.log(req.body);
-		res.sendStatus(200);
+	app.post('/posts', async function (req, res) {
+		if ('title' in req.body && 'body' in req.body) {
```

Comment:
I assume that it would be nice to separate the validation in the function. Because it can include more checks.
And call `res.sendStatus(400);` immediately after the check is failed. Just to avoid bigger nesting.